e2e/terraform/network.tf

```diff
@@ -76,3 +81,14 @@ resource "aws_security_group" "primary" {
   }
 }
 
+resource "aws_security_group" "nfs" {
+  name   = "${local.random_name}-nfs"
+  vpc_id = data.aws_vpc.default.id
+
+  ingress {
+    from_port       = 2049
+    to_port         = 2049
+    protocol        = "tcp"
+    security_groups = [aws_security_group.primary.id]
+  }
+}
```

Comment:
no opinion here, but wondering if it makes sense to move colocate this block with rest of volumes changes.  The other SG in this file seems more about exposing ports to public internet; also it may be nice to keep internal volume resources together.

Reply:
Yeah, I think having a `network.tf` file is what makes that fuzzy. If I were doing this from scratch I might consider putting the VPC and subnets under `network.tf` and then the security groups with their associated resources (`primary` in `compute.tf` and `nfs` in `volumes.tf`).